pwnlib/tubes/process.py

```diff
@@ -308,13 +304,15 @@ def __init__(self, argv = None,
             # and binfmt is not installed (e.g. when running on
             # Travis CI), re-try with qemu-XXX if we get an
             # 'Exec format error'.
-            prefixes = [([], executable)]
-            executables = [executable]
+            prefixes = [([], self.executable)]
             exception = None
 
             for prefix, executable in prefixes:
                 try:
-                    self.proc = subprocess.Popen(args = prefix + argv,
+                    args = argv
+                    if (prefix):
```

Comment:
why the parentheses?
```suggestion
                    if prefix:
```